### ISSUE TITLE
Forms missing label

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-admintools#164]
The forms from `Check security cache`, `Change Document User` and `Check user rights on page` are missing a label, which might make navigation through the form confusing for screen readers.

### Who does this affect?

People using a screen reader, including blind, low vision, and neurodivergent people
